aldjemy/core.py

```diff
@@ -80,6 +82,25 @@ def recreate(self):
         )
 
 
+def first_connect(engine, dbapi_connection, connection_record):
+    """
+    Like sqlalchemy.engine.create.<>.first_connect
+    Without rollback the transaction.
+    https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/engine/create.py#L662
```

Comment:
```suggestion
    https://github.com/sqlalchemy/sqlalchemy/blob/c2cad1f97c51c8a2a6ad5d371ece7bcd9c7ffcf9/lib/sqlalchemy/engine/create.py#L662
```
(use permalink, "y" in GitHub UI)